custom_components/econnect_metronet/devices.py

```diff
@@ -199,6 +207,8 @@ def update(self):
         # Update the internal state machine (mapping state)
         self.state = self.get_state()
 
+        return self._inventory
```

Comment:
Update the docstring to explain this method returns the inventory. Also think to return an immutable object so that nobody externally can change the internal inventory.